app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -226,58 +236,64 @@ class TopicFragmentTest {
     launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
       onView(withId(R.id.topic_tabs_viewpager)).check(matches(isDisplayed()))
       onView(withId(R.id.topic_tabs_viewpager)).perform(swipeLeft())
-      verifyTabTitleAtPosition(position = LESSON_TAB_POSITION)
+      verifyTabTitleAtPosition(position = 1)
     }
   }
 
   @Test
-  fun testTopicFragment_infoTopicTab_isDisplayedInTabLayout() {
+  fun testTopicFragment_enableExtraTabs_infoTopicTab_isDisplayedInTabLayout() {
+    enableExtraTabs()
     initializeApplicationComponent()
     launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
-      onView(withText(TopicTab.getTabForPosition(position = 0, enablePracticeTab).name)).check(
-        matches(
-          isDescendantOfA(
-            withId(
-              R.id.topic_tabs_container
-            )
-          )
+      onView(
+        withText(
+          TopicTab.getTabForPosition(
+            position = INFO_TAB_POSITION,
+            enableExtraTopicTabsUi
+          ).name
         )
-      )
+      ).check(matches(isDescendantOfA(withId(R.id.topic_tabs_container))))
     }
   }
 
   @Test
-  fun testTopicFragment_defaultTabIsInfo_isSuccessful() {
+  fun testTopicFragment_disableExtraTabs_infoTopicTab_isNotDisplayedInTabLayout() {
     initializeApplicationComponent()
     launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
-      verifyTabTitleAtPosition(position = INFO_TAB_POSITION)
+      onView(withText(TopicTab.getTabForPosition(position = INFO_TAB_POSITION, true).name))
+        .check(doesNotExist())
     }
   }
 
   @Test
-  fun testTopicFragment_defaultTabIsInfo_showsMatchingContent() {
+  fun testTopicFragment_disableExtraTabs_defaultTabIsLessons_isSuccessful() {
```

Comment:
```suggestion
  fun testTopicFragment_disableExtraTabs_defaultTabIsLessons() {
```

The 'isSuccessful' bit is redundant.

Reply:
done